### DISCUSSION TITLE
Syncing Dark Mode with the system theme

### DISCUSSION THREAD

**Discussion by mihirkumar** [darkreader/darkreader#8413] [Ideas]
I want Dark Mode to enable itself when I enable the dark system theme and turn itself off upon switching to the system light mode. I don't see a setting for this in the UI, but I do see the following in the README file:
```
// Enable when the system color scheme is dark.
DarkReader.auto({
    brightness: 100,
    contrast: 90,
    sepia: 10
});
```
So, a listener for the system theme exists as laid out above. Is there any interest in adding this feature to the UI? Is this toggle perhaps already present in the UI somewhere I'm not looking?

The new UI would have buttons for selecting between "On/Off/Auto" where "Auto" syncs dark mode with the system theme. What do you guys think?

**Comment by Gusted** [comment]
Did you try searching the issues before opening this discussion, anything related to system theme should yield some kind of issue that already point the answer.

**Reply by mihirkumar** [reply]
I had tried searching through the issues list but had not found anything conclusive. I see issues #8276, #7984, #7375 filed in the last 5 months asking for this feature as well, so I'm not alone in thinking that it could make for a good enhancement and missing the fact that it exists in the current documentation.

Seeing that it does exist, it might not be a bad idea to put a line about it in the README file.

If anyone else stumbles across this post, this feature can be enabled by:
1. Clicking on Dev tools at the bottom right of the extension window
2. Clicking on "Preview new design". An "auto" button should appear in the UI now.
3. Clicking on "Auto".

**Reply by Gusted** [reply]
> Seeing that it does exist, it might not be a bad idea to put a line about it in the README file.

With that idea we should add every option to the README, the new UI is still a prototype and all new features are only added to the new UI, the old UI also had a option for it, just a bit more hidden away for it.

**Reply by mihirkumar** [reply]
A page detailing Dark Reader's features would be a good idea. The [help](https://darkreader.org/help/en/) page can be updated to add information about syncing with the system theme.

**Reply by zarif98** [reply]
Why is this still in preview? It's been about 3 years now and seems to work fine

**Reply by Myshor** [reply]
@zarif98 Dark Reader team is small and most of us work with this project in our free time so it's not going further very fast. Specially when it goes to the new design which is still not the final one we will want to use (at least Alexander - main dev of Dark Reader consider some changes into it). That's why you can switch in settings to use new design but it's still mostly untranslated (English) in most places and not forced to use for everyone.
